Dockerfile

```diff
@@ -1,33 +1,20 @@
-FROM ubuntu:focal
+FROM python:3.10-slim
 
-WORKDIR /srv/cartography
+# the UID and GID to run cartography as
+# (https://github.com/hexops/dockerfile#do-not-use-a-uid-below-10000).
+ARG uid=10001
+ARG gid=10001
 
-ENV PATH=/venv/bin:$PATH
-RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.10-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev curl make git && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+COPY . /var/cartography
+WORKDIR /var/cartography
+ENV HOME=/var/cartography
 
-# Installs pip supported by python3.10
-RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3.10 get-pip.py
+RUN pip install -U -e .
```

Comment:
Production install should not use `--editable`.

Reply:
This is the way it already is right now though: https://github.com/lyft/cartography/blob/master/dist.Dockerfile

The other one that I removed is this way too: https://github.com/lyft/cartography/blob/6a6e2778204938cbd51add75827f7aaacef31526/Dockerfile#L20

Reply:
Are you saying that we should make it 

```Dockerfile
RUN pip install .
```

instead? I think that works.

Reply:
```pip install cartography```

Docker steps should be the exact commands that we will use to setup a production environment on a bare Linux host, it is just that the image has everything the environment made up of such as Python, Cartography package installed with all its production dependencies only (no package that are required for dev/test).

Reply:
Mm, maybe I will add back the dev.Dockerfile then.

dev.Dockerfile will run an editable install

Dockerfile will install the latest release